v3/src/components/slider/inspector-panel/slider-settings-panel.tsx

```diff
@@ -46,7 +46,7 @@ export const SliderSettingsPalette =
           <Flex className="palette-row">
             <FormLabel className="form-label">{t("DG.Slider.multiples")}
               <NumberInput className="slider-input multiples" size="xs" defaultValue={sliderModel.multipleOf}
-                  min={0} step={0.5} onChange={handleMultiplesOfChange}>
+                  min={0.5} step={0.5} max={sliderModel.axis.max} onChange={handleMultiplesOfChange}>
```

Comment:
I don't think the "multiples" input should have a minimum or maximum. V2 doesn't. You may want tiny increments if the axis is way zoomed in. And there isn't any reason to restrict the user's desire for a max either. I think the same holds for the max frames per second field, but for some reason I'm not able to leave a comment there.